docs/contributing/how-to-release.md

```diff
@@ -67,7 +62,7 @@ The following files need to be uploaded:
     wxWidgets-x.y.z.7z
     wxWidgets-x.y.z.tar.bz2
     wxWidgets-x.y.z.zip
-    wxWidgets-x.y.z-docs-chm.zip
+    wxWidgets-x.y.z-docs.chm
```

Comment:
The only reason to ZIP it that I see is that some paranoid IT departments forbid download of all executable files and CHM is considered to be executable (probably for a good reason, too). OTOH such places usually also scan ZIP files for executables anyhow as well...

Anyhow, let's try not ZIPping them and we can always change back if anybody complains.